common/web/keyboard-processor/src/text/codes.ts

```diff
@@ -1,31 +1,36 @@
-// TODO:  Move to separate folder:  'codes'
-// We should start splitting off code needed by keyboards even without a KeyboardProcessor active.
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Keyboard key codes and modifier bitmasks.
+ */
 
-// see also: common/web/types/src/kmx/kmx.ts
+import { ModifierKeyConstants, USVirtualKeyCodes } from '@keymanapp/common-types';
 
 const Codes = {
-  // Define Keyman Developer modifier bit-flags (exposed for use by other modules)
-  // Compare against /common/include/kmx_file.h.  CTRL+F "#define LCTRLFLAG" to find the secton.
   modifierCodes: {
-    "LCTRL":0x0001,           // LCTRLFLAG
-    "RCTRL":0x0002,           // RCTRLFLAG
-    "LALT":0x0004,            // LALTFLAG
-    "RALT":0x0008,            // RALTFLAG
-    "SHIFT":0x0010,           // K_SHIFTFLAG
-    "CTRL":0x0020,            // K_CTRLFLAG
-    "ALT":0x0040,             // K_ALTFLAG
+    ...ModifierKeyConstants,
```

Comment:
I don't think we should include this here -- we should maintain this as a legacy interface only. Mixing the names in a published interface doesn't buy us anything -- we have to maintain the old names anyway.

Reply:
Done.